registry.json

```diff
@@ -1,4 +1,19 @@
 [
+      {
+      "ticker":"ICRYP",
+      "poolId":"2671702f26f947ecc114861c4983bfdc8616d3fa2bace18dd4a91e65",
+      "social":{
+         "facebook":"",
+         "twitter":"icrypnet",
+         "website":"https://www.icryp.net",
+         "reddit":"",
+         "instagram":"",
+         "youtube":"The Last Generation"
+      },
+      "telegram":"",
```

Comment:
We need a personal telegram handle(no channel or group) this is for voting.

Reply:
I already changed it.. Did it show the changed?